SUMO-227092: Support terraform for azure event hub log source (#590)
SumoLogic/terraform-provider-sumologic: website/docs/r/polling_source.html.markdown

layout: "sumologic"
page_title: "SumoLogic: sumologic_polling_source"
description: |-

>   Provides a Sumologic Polling source. This source is used to import data from various AWS and Azure products.

Reviewer: No need to edit this as this source is deprecated already.

>   Provides a Sumologic Polling source. This source is used to import data from various AWS products as well as Azure Event Hub.

Reviewer 2: I know this is the first Azure product, but from doc perspective maybe this is cleaner? What do you think?
```suggestion
  Provides a Sumologic Polling source. This source is used to import data from various AWS and Azure products.
```

Author: Sounds reasonable.
